web/models/auth_handler.ex

```diff
@@ -6,10 +6,12 @@ defmodule EspiDni.AuthHandler do
   alias Ueberauth.Auth
   alias EspiDni.TeamFromAuth
   alias EspiDni.UserFromAuth
+  import EspiDni.Gettext
 
   def init_from_auth(%Auth{} = auth) do
     with {:ok, team} <- TeamFromAuth.find_or_create(auth),
-         {:ok, user} <- UserFromAuth.find_or_create(auth, team) do
+         {:ok, user} <- UserFromAuth.find_or_create(auth, team),
+         {:ok, user} <- send_welcome_message(user) do
       {:ok, team, user}
     end
     |> case do
```

Comment:
Is this a common way to do the `when`? Looks a bit hard to read whats going on. Could the `case` part be moved into a private function?